packages/kbn-cases-components/src/tooltip/tooltip.tsx

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React, { memo } from 'react';
+import { EuiToolTip } from '@elastic/eui';
+
+import { TooltipContent } from './tooltip_content';
+import type { CaseTooltipProps } from './types';
+import { Skeleton } from './skeleton';
+
+interface Props extends CaseTooltipProps {
```

Comment:
I think we should a) rename `Props` to `CaseTooltipProps` b) rename `CaseTooltipProps` to `CaseTooltipContentProps` and export the props of the tooltip and the props of the content. Otherwise, the consumer of the component cannot type correctly the props (like loading etc) that will be passed to our tooltip. Also, maybe it makes sense to separate the props of the tooltip from the content like

```
interface CaseTooltipProps {
  children: React.ReactNode;
  content: CaseTooltipContentProps
  dataTestSubj?: string;
  className?: string;
  loading?: boolean;
}
```